ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -15,53 +15,76 @@ class ButtonDemoController: DemoController {
-            disabledButton.setTitle("Button", for: .normal)
-
-            addRow(items: [button, disabledButton], itemSpacing: 20)
+            let button = makeButton(style: style,
+                                    title: "Button")
+            let disabledButton = makeButton(style: style,
+                                            title: "Button",
+                                            isEnabled: false)
+            let titleButtonStack = UIStackView(arrangedSubviews: [button, disabledButton])
+            titleButtonStack.spacing = 20
+            titleButtonStack.distribution = .fillProportionally
+            container.addArrangedSubview(titleButtonStack)
 
             if let image = style.image {
-                let iconButton = Button(style: style)
-                iconButton.setTitle("Button", for: .normal)
-                iconButton.image = image
-
-                let disabledIconButton = Button(style: style)
-                disabledIconButton.isEnabled = false
-                disabledIconButton.setTitle("Button", for: .normal)
-                disabledIconButton.image = image
-
-                addRow(items: [iconButton, disabledIconButton], itemSpacing: 20)
-
-                let iconOnlyButton = Button(style: style)
-                iconOnlyButton.image = image
-
-                let disabledIconOnlyButton = Button(style: style)
-                disabledIconOnlyButton.isEnabled = false
-                disabledIconOnlyButton.image = image
-
-                addRow(items: [iconOnlyButton, disabledIconOnlyButton], itemSpacing: 20)
+                let iconButton = makeButton(style: style,
+                                            title: "Button",
+                                            image: image)
+                let disabledIconButton = makeButton(style: style,
+                                                    title: "Button",
+                                                    image: image,
+                                                    isEnabled: false)
+                let titleImageButtonStack = UIStackView(arrangedSubviews: [iconButton, disabledIconButton])
+                titleImageButtonStack.spacing = 20
+                titleImageButtonStack.distribution = .fillProportionally
+                container.addArrangedSubview(titleImageButtonStack)
+
+                let iconOnlyButton = makeButton(style: style,
+                                                image: image)
+                let disabledIconOnlyButton = makeButton(style: style,
+                                                        image: image,
+                                                        isEnabled: false)
+                let imageButtonStack = UIStackView(arrangedSubviews: [iconOnlyButton, disabledIconOnlyButton])
+                imageButtonStack.spacing = 20
+                imageButtonStack.distribution = .fillProportionally
+                container.addArrangedSubview(imageButtonStack)
```

Comment:
Probably not, considering addRow is something custom to our base demo controller, that comes with lots of odd configuration choices and lots of extra stacks.

Reply:
The root issue of too deeply nested of stacks might be worth mentioning, just because people use the demo controllers as examples for implementation. But not a huge deal either way.